dips/dip-196.md

```diff
@@ -0,0 +1,70 @@
+---
+dip: 196
+title: Onchain Network Identity
+authors: Greg Nazario (@gregnazario)
+status: Draft
+type: Informational
+created: 07/06/2021
+last_updated: 07/06/2021
+issue: https://github.com/diem/dip/issues/196
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+# Simple Summary
+
+This DIP describes Onchain Network, an identity for Diem nodes to easily discover and update identity of other Diem nodes on the network.  Allows for trusted connections between nodes.
```

Comment:
Hmm.... This sentence seems a bit lacking:

This DIP describes Onchain Network Identities, a framework for Diem nodes to easily discover and update network credentials and permissions for other nodes on the network. Examples of such permissions include priority or privileged access for well-known participants, such as VASPs, to upstream validator fullnodes. This becomes increasingly important as a means to mitigate resource limitations of the validator fullnodes, where the validator fullnodes cannot support the demands of the entire ecosystem but can still support nodes from entities critical for network operations. While in theory this can be handled without the blockchain leveraging an identity repository and configuration file manipulation, the requirements for scaling and securing such a solution would be substantially more operational overhead and duplicate functionality already provided by the Diem blockchain.